packages/flutter/test/material/radio_list_tile_test.dart

```diff
@@ -1336,4 +1334,114 @@ void main() {
       expect(feedback.hapticCount, 0);
     });
   });
+
+  group('Material 2', () {
```

Comment:
I think I also use `material3` to check the expected behavior in test 'RadioListTile respects hoverColor'. If we want to put all M2 tests in a group, I think we might also need to move it here:)

Reply:
That sounds good!